storm-core/src/jvm/org/apache/storm/utils/Utils.java

```diff
@@ -1374,5 +1393,21 @@ public static RuntimeException wrapInRuntime(Exception e){
             return new RuntimeException(e);
         }
     }
+
+    public static int getAvailablePort(int port) {
```

Comment:
will be good to rename the port to preferredPort since it only makes a best effort guarantee.